lib/internal/util/colors.js

```diff
@@ -24,18 +24,16 @@ module.exports = {
         stream.getColorDepth() > 2 : true);
   },
   refresh() {
-    if (process.stderr.isTTY) {
-      const hasColors = module.exports.shouldColorize(process.stderr);
-      module.exports.blue = hasColors ? '\u001b[34m' : '';
-      module.exports.green = hasColors ? '\u001b[32m' : '';
-      module.exports.white = hasColors ? '\u001b[39m' : '';
-      module.exports.yellow = hasColors ? '\u001b[33m' : '';
-      module.exports.red = hasColors ? '\u001b[31m' : '';
-      module.exports.gray = hasColors ? '\u001b[90m' : '';
-      module.exports.clear = hasColors ? '\u001bc' : '';
-      module.exports.reset = hasColors ? '\u001b[0m' : '';
-      module.exports.hasColors = hasColors;
-    }
+    const hasColors = module.exports.shouldColorize(process.stderr);
+    module.exports.blue = hasColors ? '\u001b[34m' : '';
+    module.exports.green = hasColors ? '\u001b[32m' : '';
+    module.exports.white = hasColors ? '\u001b[39m' : '';
+    module.exports.yellow = hasColors ? '\u001b[33m' : '';
+    module.exports.red = hasColors ? '\u001b[31m' : '';
+    module.exports.gray = hasColors ? '\u001b[90m' : '';
+    module.exports.clear = hasColors ? '\u001bc' : '';
+    module.exports.reset = hasColors ? '\u001b[0m' : '';
+    module.exports.hasColors = hasColors;
```

Comment:
Non-blocking and admittedly more verbose (and I know not introduced in this PR) but I'd generally prefer something like the following instead. 
```suggestion
    if (module.exports.shouldColorize(process.stderr)) {
      module.exports.blue = '\u001b[34m';
      module.exports.green = '\u001b[32m';
      module.exports.white = '\u001b[39m';
      module.exports.yellow = '\u001b[33m';
      module.exports.red = '\u001b[31m';
      module.exports.gray = '\u001b[90m';
      module.exports.clear = '\u001bc';
      module.exports.reset = '\u001b[0m';
      module.exports.hasColors = true;
    } else {
      module.exports.blue = '';
      module.exports.green = ';
      module.exports.white = ';
      module.exports.yellow = ';
      module.exports.red = ';
      module.exports.gray = '';
      module.exports.clear = '';
      module.exports.reset = '';
      module.exports.hasColors = false;
    }
```

Reply:
Hey @jasnell, thanks for the review.  
I don't have strong opinions on this specific case, but in general, I think the more readable, the better.

We would have some repetitions, but I think it would be worth it.

Up to you 🚀  If you want, I can use this PR to address this readability issue as well 😁

Reply:
the CI is green now, maybe consider doing a follow up PR.

Reply:
I agree, also because I have the impression that osx CI has been a little unstable in the last two days